kdevnel/ynab-absa-csv-convert-js
# Reorganise modules and general improvements

## Description

* Moved file upload event handler to a separate module - closes #31
* Moved a few callbacks to separate functions to improve code formatting
* Process memo column data in the same way as other columns in modifyValues() - it's now part of the switch statement - closes #29
* Refactor to improve function and variable naming across the project - closes #30 
* A few other minor improvements - closes #28, closes #5

## Testing
* Run `npm install` then `npm start` from the root folder to test in development mode.

OR

* Open `/dist/index.html` in a browser

You can then use `/testing/input-sample.csv` and you should see an array of data in the console, a table in the document with the processed data and the output CSV file can be downloaded.
